openmls/src/ciphersuite/signature.rs

```diff
@@ -36,7 +36,7 @@ pub struct SignContent {
     content: VLBytes,
 }
 
-const SIGN_LABEL_PREFIX: &str = "MLS 1.0";
+const SIGN_LABEL_PREFIX: &str = "MLS 1.0 ";
```

Comment:
Can't we have this label at one place? E.g. in `ciphersuite/mod.rs`. Right now it's the same in `hpke` and `signature`.